script/installation/packages.sh

```diff
@@ -12,9 +12,8 @@
 ## g++ manually.
 ##
 ## Supported environments:
-##  * Ubuntu (14.04, 16.04)
-##  * Fedora
-##  * OSX
+##  * Ubuntu (14.04, 16.04, 18.04)
+##  * macOS
```

Comment:
I would add a message here:

```
Update (2018-06-08):
We are no longer able to support RedHat/Fedora because those environments are not supported by TensorFlow.
```

Reply:
Added.